src/components/landingPage/components/LandingPage.js

```diff
@@ -67,50 +67,53 @@ const LandingPage = () => {
+                    <div className="homepage-wrapper__content__side-bar">
+                        {
+                            MENUITEMS.map(menuitem => (
+                                <div
+                                    aria-label="Mute volume"
+                                    onClick={() => changeCurrentText(menuitem.index)}
+                                    className={`item ${(menuitem.index === currentText.index) ? 'active' : ''}`}
+                                    key={`${menuitem.FIRSTTITLE}-${menuitem.index}`}
+                                    onKeyDown={dummy}
+                                    role="button"
+                                    tabIndex="0"
+                                />
+                            ))
+                        }
+                    </div>
+                    <div className="homepage-wrapper__content--wrapper --animated" key={stateKey}>
+                        <div className="homepage-wrapper__content__description">{currentText.FIRSTTITLE}</div>
+                        <div className="homepage-wrapper__content__description --bolder">{currentText.SECONDTITLE}</div>
+                        <div className="homepage-wrapper__content__text">
+                            {currentText.HOMETEXT}
+                        </div>
                     </div>
                 </div>
-            </div>
-            {/* content for displaying on the home page */}
+                {/* content for displaying on the home page */}
 
-            {/* footer for the homepage */}
-            <div className="homepage-wrapper__footer">
+                {/* footer for the homepage */}
+                <div className="homepage-wrapper__footer">
 
-                <a href={FACEBOOK_PAGE}>
-                    <img className="--small" src={Facebook} alt="Facebook" />
-                </a>
-                <a href={INSTAGRAM_PAGE}>
-                    <img className="--small" src={Instagram} alt="Instagram" />
-                </a>
-                <a href={TWITTER_PAGE}>
-                    <img className="--small" src={Twitter} alt="Twitter" />
-                </a>
-                <a href={YOUTUBE_PAGE}>
-                    <img className="--small" src={Youtube} alt="Youtube" />
-                </a>
+                    <a href={FACEBOOK_PAGE}>
+                        <img className="--small" src={Facebook} alt="Facebook" />
+                    </a>
+                    <a href={INSTAGRAM_PAGE}>
+                        <img className="--small" src={Instagram} alt="Instagram" />
+                    </a>
+                    <a href={TWITTER_PAGE}>
+                        <img className="--small" src={Twitter} alt="Twitter" />
+                    </a>
+                    <a href={YOUTUBE_PAGE}>
+                        <img className="--small" src={Youtube} alt="Youtube" />
+                    </a>
```

Comment:
don't do this, move this constants to array of objects and map over it.
also if this is a reusable footer, consider making a component

Reply:
i have corrected it in the next two PR's i am going to make